pyproject.toml

```diff
@@ -45,7 +44,6 @@ keywords = [
 license = {text = "Apache Software License (Apache 2.0)"}
 classifiers = [
     "Development Status :: 4 - Beta",
-    "Programming Language :: Python :: 3.8",
```

Comment:
Again not technically part of the ticket, but 3.12 is missing here.

Reply:
Also done.